UnitySetup/UnitySetup.psm1

```diff
@@ -1870,7 +1870,22 @@ function Start-UnityEditor {
             $process.Start() | Out-Null
 
             if ( $Wait ) {
-                $process.WaitForExit()
+                if( $LogFile ) {
+                    New-Item -Path $LogFile -ItemType File -Force
```

Comment:
Unity should be creating this file so it's a little scary to force create here. Can the job wait for the file to exist then start getting content?

Reply:
I've been using this in my automated builds. Doesn't seem to hurt anything if we create the file first.

Reply:
It's a bit of a race condition between the script and Unity so it really scares me - you could even end up overwriting logs. Better I think to poll (with a timeout maybe?) in the job waiting for the file to exist then call `Get-Content -Wait`

Reply:
haha, that's fair. I have no idea how to begin doing something like that.

Reply:
The job could loop on testing `Test-Path` until the file exists I think. Since we can't wait on the job anyways I don't think the timeout is necessary.

Reply:
Could you write this up real quick? I'm not sure how to test the path.

Reply:
nvm I got it.